### ISSUE TITLE
AttributeError: module 'tomopy' has no attribute 'io'

### ISSUE THREAD

**Issue by aaronclaeys** [tomopy/tomopy#471]
Hello,

I followed advice given to me in #470 and I updated Python to 3.8 and TomoPy to 1.7.1 and now the reconstruction script will run until reconstruction is complete but it now produces the following error:

```python

Traceback (most recent call last):
  File "Aaron1.py", line 502, in <module>
    loopEngine(data_files[ii],output_files[ii],
  File "Aaron1.py", line 325, in loopEngine
    tomopy.io.writer.write_tiff_stack(data_recon[np.int(margin_slices/2):(sliceEnd-sliceStart-np.int(margin_slices/2)),:,:], 
AttributeError: module 'tomopy' has no attribute 'io'
```
I'm not sure what to do or how to troubleshoot this error. Any help is appreciated, thanks.

**Platform Information:**
 - OS: Linux
 - Python 3.8
 - TomoPy 1.7.1

**Additional context**
Aaron1.py is the reconstruction script

**Comment by carterbox** [comment]
Hi @aaronclaeys,

tl;dr `write_tiff_stack` was removed from tomopy in version 1.1. Modify your script to import `write_tiff_stack` from the [dxchange](https://dxchange.readthedocs.io/en/latest/source/api/dxchange.writer.html) package or use another library like `tifffile`.

Python packages may be organized into nested modules, so that all the code isn't in one huge file. If you look at our [API reference](https://tomopy.readthedocs.io/en/1.7.1/api.html), you can see a list of our current modules. That error means that `tomopy` has no sub-module named `io`. Your code is trying to load the `write_tiff_stack` function from the `tomopy.io` module which does not exist. It looks like all the read/write functions were moved out of TomoPy into the DXchange package in version 1.1. Apparently this change wasn't properly deprecated; we were a less vigilant about versioning in 2017.

Hope that helps!
